### DISCUSSION TITLE
DNS /NAME SERVER

### DISCUSSION THREAD

**Discussion by NYCEngineer** [planetscale/discussion#500] [Q&A] [answered]
Newbie here. Database is already uploaded yet can't go live because name server is missing. 

Anybody know what it is?

Thank you in advance.

**Comment by jgreet** [comment, marked as answer]
I think we addressed this in a support ticket but the host name you can use for any branch is `aws.connect.psdb.cloud` or `gcp.connect.psdb.cloud`, depending on the region selected for the branch.
